### PR TITLE
[ML4SE-228] Agreements saves into the special json file.

### PR DESCRIPTION
**Pull request by mikrise2**

### CODE REVIEW

**Review by eartser — CHANGES_REQUESTED** [review]
Great, thanks! Left only a couple of small comments

File: ijPlugin/src/main/resources/org/jetbrains/research/tasktracker/ui/main/panel/template/agreement.html [hunk 1]
```diff
@@ -18,7 +18,6 @@
 <script>
     let requiredFields = document.querySelectorAll('input[required]')
 
-    // Loop through each required input element and add the 'required' class to its parent label
     requiredFields.forEach(function (inputElement) {
```
[the remark this answers is not in the document]
Comment (mikrise2) [reply ijPlugin/src/main/resources/org/jetbrains/research/tasktracker/ui/main/panel/template/agreement.html:21]: Unfortunately, our architecture only allows for easily adding CSS files to HTML at the moment, and it seems like it's sufficient for now. However, we can open this question up for discussion. What do you think?

[the remark this answers is not in the document]
Comment (eartser) [reply ijPlugin/src/main/resources/org/jetbrains/research/tasktracker/ui/main/panel/template/agreement.html:21]: I see, then I suggest leaving this as an idea for a future refactoring

File: ijPlugin/src/main/resources/org/jetbrains/research/tasktracker/ui/main/panel/template/agreement.html [hunk 2]
```diff
@@ -35,7 +35,7 @@
         };
     });
 
-    function allChecked() {
+    function check_all_inputs() {
         requiredFields.forEach(function (field) {
```
Comment (eartser) [inline ijPlugin/src/main/resources/org/jetbrains/research/tasktracker/ui/main/panel/template/agreement.html:39]: I guess this name should be in camel case as well

**Review by eartser — APPROVED** [review]

**Pull request merged**